web/app.js

```diff
@@ -1641,6 +1642,23 @@ function webViewerInitialized() {
         fileInput: evt.target,
       });
     });
```

Comment:
Nit: Let's add a new line here, to improve readability since the code above has nothing to do with drag and drop.